exp/views/study.py

```diff
@@ -9,12 +9,7 @@
 from django.db.models import Q
 from django.db.models.functions import Lower
 from django.forms.models import BaseModelForm
-from django.http import (
-    HttpRequest,
-    HttpResponse,
-    HttpResponseForbidden,
-    HttpResponseRedirect,
-)
+from django.http import HttpRequest, HttpResponseForbidden, HttpResponseRedirect
 from django.http.response import HttpResponse
```

Comment:
I think `HttpResponse` was removed here because it's also imported below from `django.http.response`. But it looks to me like it should be imported from `django.http`, not `django.http.response` (see [docs here](https://docs.djangoproject.com/en/4.2/ref/request-response/#django.http.HttpResponse)). 
Should we change this to:
```python
from django.http import HttpRequest, HttpResponse, HttpResponseForbidden, HttpResponseRedirect
```
And remove this line:
```python
from django.http.response import HttpResponse
```

Reply:
Per the docs, `django.http` is the correct import, but both probably work. https://docs.djangoproject.com/en/5.0/ref/request-response/#usage

Reply:
Fixed in commit https://github.com/lookit/lookit-api/pull/1397/commits/38b12e47771da372ad8ec5f16e5c5a864b924185.